classes/04class/exercises/c04-iac01/nimmi89/security-group.tf

```diff
@@ -0,0 +1,14 @@
+resource "aws_security_group" "DA-sg"{
+    ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
For security, use your home ip address here.

Reply:
Yes. Missed that.I have updated the same